kivymd/toast/kivytoast/kivytoast.py

```diff
@@ -77,7 +77,6 @@ class Toast(BaseDialog):
     and defaults to `2.5`.
     """
 
-    _md_bg_color = ListProperty()
```

Comment:
Why would you remove this attribute if it is in use?

```python
def toast(text="", background=None, duration=2.5):
    """Displays a toast.

    :attr duration: the amount of time (in seconds) that the toast is visible on the screen
    :type duration: float

    :attr background: color ``rgba`` in Kivy format
    :type background: list
    """

    if not background:
        background = [0.2, 0.2, 0.2, 1]
    Toast(duration=duration, _md_bg_color=background).toast(text)
````

Reply:
Because user may confuse with the args in `android` and `kivytoast` (`color` arg that passed to the `toast` is only works on `kivytoast` not on `android` toast).

Reply:
You should have suggested an alternative and not remove the `_md_bg_color` attribute

Reply:
Reverted it.